geniza/corpus/templates/admin/corpus/document/change_form.html

```diff
@@ -18,11 +18,21 @@
         </fieldset>
     {% endif %}
     {% if original.has_transcription %}
-        <h2>Transcription</h2>
-        {% for footnote in original.footnotes.all %}
-            {% if footnote.content %}
-                {% include "footnotes/transcription.html" with transcription=footnote.content %}
-            {% endif %}
-        {% endfor %}
+        <fieldset class="module aligned transcriptions-field">
+            <div class="form-row">
+                <label>Transcription(s)</label>
+                {% regroup original.footnotes.all by source as source_list %}
+                {% for source in source_list %}
+                    {% if source_list|length > 1 %}<h3 class="source">{{ source.grouper|safe }}</h3>{% endif %}
+                    {% for footnote in source.list %}
+                        {% if footnote.content %}
+                            {% include "footnotes/transcription.html" with transcription=footnote.content %}
+                        {% else %}
+                            <div class="transcription">No transcription</div>
```

Comment:
I don't think we should display this here — it's redundant and takes up space.

There's already a digitized transcription red/green boolean indicator in the footnote inline display; if that isn't enough for the team (in combination with labels for the transcriptions), then we should figure out what would make it more readable for them.

Reply:
Done in c62ff7a 👍